fcrepo-http-api/src/test/java/org/fcrepo/http/api/repository/FedoraRepositoryNodeTypesTest.java

```diff
@@ -48,6 +48,7 @@
  *
  * @author cbeer
  */
+@Deprecated
```

Comment:
We should instead add this update to its own PR/ticket... related to:
https://jira.duraspace.org/browse/FCREPO-1890